.appveyor.yml

```diff
@@ -91,6 +89,7 @@ install:
   - echo %PYTHON_VERSION% %TARGET_ARCH%
   # pytest-cov>=2.3.1 due to https://github.com/pytest-dev/pytest-cov/issues/124
   - pip install -q pytest "pytest-cov>=2.3.1" pytest-rerunfailures pytest-timeout pytest-xdist
+    cycler pyparsing backports.functools-lru-cache
```

Comment:
Putting these in this "quiet" pip section should shave a few hundred useless lines from the output log.

Reply:
These packages will be installed automatically (#9673).

Reply:
Hmm, but they're installed at the same time as `Matplotlib`, which I don't think we want to have a quiet flag. Putting them here prevents pip spewing out a load of extra lines that aren't very useful for debugging.